lib/usdUfe/ufe/UsdClipboard.cpp

```diff
@@ -21,13 +21,43 @@
 #include <pxr/usd/usd/usdcFileFormat.h>
 #include <pxr/usd/usdShade/connectableAPI.h>
 
+#include <ufe/globalSelection.h>
+#include <ufe/observableSelection.h>
+#include <ufe/observer.h>
+#include <ufe/selectionNotification.h>
+
 #if (__cplusplus < 201703L)
 #error "Compiling ClipboardHandler requires C++17"
 #endif
 #include <filesystem> // Requires C++17
 
 namespace USDUFE_NS_DEF {
 
+// TufeSelectionObserver
+//
+class UsdClipboard::TufeSelectionObserver : public Ufe::Observer
+{
+public:
+    TufeSelectionObserver(UsdClipboard& cb)
+        : Ufe::Observer()
+        , _clipboard(cb)
+    {
+    }
+
+    void operator()(const Ufe::Notification& notif)
+    {
+        // EMSUSD-1122 - As a user, I'd like to copy and then paste a prim as a sibling
+        // If the selection hasn't changed between the copy and the paste, then we'll
+        // paste into the parent of each destination parent item.
+        if (dynamic_cast<const Ufe::SelectionChanged*>(&notif)) {
+            _clipboard.ufeSelectionChanged();
+        }
+    }
```

Comment:
Used to track if there is a selection change between the copy and paste commands.